save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -0,0 +1,56 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.utils.EnumType
+import com.saveourtool.save.utils.LocalDateTime
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name organization
+ * @property status
+ * @property startTime the time contest starts
+ * @property endTime the time contest ends
+ * @property description
+ */
+@Entity
+@Serializable
+@Suppress("USE_DATA_CLASS")
+data class Contest(
+    var name: String,
```

Comment:
Should we also add category similar to benchmarks?

Reply:
I guess we will need it later, but for now it seems ok